soc/nxp/imxrt/Kconfig

```diff
@@ -113,9 +113,10 @@ config NXP_IMX_EXTERNAL_HYPERRAM
 	  This setting should be enabled when the application uses HYPERRAM,
 	  or an MPU region will be defined to disable cached access to the
 	  HYPERRAM memory space.
+
 config SECOND_CORE_MCUX
 	bool "Dual core operation on the RT11xx series"
-	depends on SOC_SERIES_IMXRT11XX
+	depends on SOC_SERIES_IMXRT11XX || SOC_SERIES_IMXRT118X
```

Comment:
How is IMXRT118x any differenct from IMXRT11XX?

Reply:
It is already separated into separate folder `imxrt118x`:
https://github.com/zephyrproject-rtos/zephyr/tree/main/soc/nxp/imxrt/imxrt118x
not sure why it is not in `imxrt11xx` it is then bit confusing.
https://github.com/zephyrproject-rtos/zephyr/tree/main/soc/nxp/imxrt/imxrt11xx
So I had to add `|| SOC_SERIES_IMXRT118X`

Reply:
I guess what I meant was that the SOC_SERIES_IMXRT11XX probably should include the IMXRT118x parts